ci-operator/step-registry/cucushift/installer/check/azure/private/cucushift-installer-check-azure-private-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: cucushift-installer-check-azure-private
+  steps:
+  - ref: cucushift-installer-check-azure-private-lb-loadbalancer
+  - ref: cucushift-installer-check-azure-storage-account
```

Comment:
if in the future, we are going to spread the step for every azure cluster, on that time, we need to move the step out to some common check chain, right?

Reply:
correct, if we have common check chain on all azure cluster including this step, then need to remove it form here.